### ISSUE TITLE
Gtk3-demo-application.exe complains with "No Disk"

### ISSUE THREAD

**Issue by JESteward** [msys2/MINGW-packages#5785]
With both 32 and 64 bit Gtk+3.0 demo applications, if the app is run by double clicking the exe or running it from cmd.exe started outside an MSYS2 shell, when you go to open a file a message window pops up titled gtk3-demo-application.exe - No Disk, and with the content "There is no disk in the drive. Please insert a disk into the drive \Device\Harddisk1\DR1."  Note that the hard disk number and DR number may vary on other users' systems.
When the demo app is run from an MSYS2 prompt (or MinGW32 or MinGW64), there is no pop up message window.
Is this a problem with my MSYS2/MinGW 32/MinGW 64 installation, or a Gtk bug perhaps?  Is there a work around so that I can build and deploy a Gtk3.0 app that isn't dependent on being launched from an MSYS2 shell?

**Comment by lazka** [comment]
Likely something related to https://gitlab.gnome.org/GNOME/gtk/issues/418. It reads from all drives and some card readers on Windows will error out.

**Comment by JESteward** [comment]
Yes I think it is related.  I've read dozens of such bug reports in an attempt to find a solution.  The question is why the problem does not occur when the app is run from an MSYS2/MinGW prompt?  Does the MSYS2/MinGW environment change the behaviour of Gtk?  

I wonder, does the problem occur with Gtk libs built for use with MSVC?  Does the problem only occur for apps built using the MSYS2 Gtk libs when the app is run outside of an MSYS2/MinGW environment?
